lib/registrant.rb

```diff
@@ -0,0 +1,19 @@
+class Registrant
+attr_reader :name, :age, :permit, :license_data
+
+  def initialize(name, age, permit = false)
+    @name = name
+    @age = age
+    @permit = permit
+    @license_data = {:written=>false, :license=>false, :renewed=>false}
+  end
+
+  def permit?
+    @permit
+  end
+
+  def earn_permit
+    @permit = true
+  end
+
+end
```

Comment:
👍 This looks good, however there is no need to list `:permit` as an attr_Reader since we never need to call `.permit` on a `Registrant` object.